OutOfSchool/OutOfSchool.WebApi.Tests/Services/ValidationServiceTests.cs

```diff
@@ -72,15 +72,18 @@ public async Task UserIsProviderOwnerAsync_WhenFalse_ReturnsFalse()
         }
 
         [Test]
-        public void UserIsProviderOwnerAsync_WhenEntityWasNotFaund_ThrowsException()
+        public async Task UserIsProviderOwnerAsync_WhenEntityWasNotFaund_ReturnsFalse()
```

Comment:
mistype `Faund` - should be `Found`. I see that it's not your change, but would be good to fix it as well )